### ISSUE TITLE
Missing Alt text on Logo link

### ISSUE THREAD

**Issue by phucduong86** [phucduong86/phucduong86.github.io#5]
![2018-11-07_1540](https://user-images.githubusercontent.com/21692768/48159550-84c74a80-e2a3-11e8-92f5-192e6fa26da9.png)

**Comment by phucduong86** [comment]
Issue resolved.
